install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# This is an install script for notes
+
+function assertInstalled() {
```

Comment:
This can be reproduced with `cat install.sh | bash` and `cat install.sh | sh` as well, which means there's something fundamentally wrong with how I'm making a function.